### ISSUE TITLE
Font size and position wrong with enabled Privacy Badger

### ISSUE THREAD

**Issue by janmanek** [EFForg/privacybadger#2556]
Found on site https://www.tradingview.com 

You can notice that font size and position of the current position in chart is wrong.
I also tried to add that site do Disabled Sites but without success.

Note: You must refresh page after enabling/disabling Privacy Badger.

Found on Firefox 73.0, Windows 10, Privacy Badger 2020.1.13

![image](https://user-images.githubusercontent.com/25988687/74683423-37859880-51c9-11ea-82b6-7efebdfaa20c.png)

**Comment by ghostwords** [comment]
Hello and thanks for the report! Looks like #1854 is still an issue.

>I also tried to add that site do Disabled Sites but without success.

Have you tried using the "Disable Privacy Badger for this site" button in Privacy Badger popup?

**Comment by janmanek** [comment]
Yeah, that helped. I didn't include "www." when I added it manually.

**Comment by ghostwords** [comment]
We'll try to fix or at least work around the underlying issue in the new update or two.

Could you tell me a bit more about trying to disable Privacy Badger on this site? Why did you try via the options page instead of the "disable" button in Badger's popup?

**Comment by ghostwords** [comment]
@janmanek I'm trying to figure out if we need to make the "Disable Privacy Badger for this site" button more prominent somehow. I would appreciate any insight you could provide into how you went about disabling Privacy Badger.

**Comment by janmanek** [comment]
Honestly, I don't know. When I noticed that issue I was at first trying to find what's wrong using Style Inspector, then I tried to disable all extensions. And then I was enabling extensions one by one and looking for the culprit. When I found it I tried to look at the extension configuration first located under Add-ons / Options. The button under Privacy Badger button doesn't seem to be somehow "hidden" or "invisible" for me. However when I look at a similar feature Bitdefender Anti-tracker provides, it seems to be a little bit more "visible".
